libandroid-navigation/src/test/java/com/mapbox/services/android/navigation/v5/navigation/NavigationWalkingOptionsTest.java

```diff
@@ -0,0 +1,44 @@
+package com.mapbox.services.android.navigation.v5.navigation;
+
+import com.mapbox.api.directions.v5.WalkingOptions;
+
+import org.junit.Test;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+public class NavigationWalkingOptionsTest {
```

Comment:
Not a big fan of "testing setters" - especially here that we're using AutoValue for creating the `WalkingOptions`.
In any case, there's this getter https://github.com/mapbox/mapbox-navigation-android/blob/42a157a54f996a350194a06b7f7d276c979a9b5d/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationWalkingOptions.java#L12 available so we don't need Mockito at all and we can assert _state_ instead of _behavior_. What do you think? Does that make sense?